ansible/roles/hub-containers/tasks/main.yml

```diff
@@ -0,0 +1,95 @@
+- name: copy over decomposed script
+  copy:
+    src: docker-decomposed.sh
+    dest: /usr/local/bin/docker-decomposed.sh
+    mode: 0755
+    owner: root
+    group: root
+  notify:
+    - redeploy containers
+
+- name: install deployment and tooling scripts
+  template:
+    src: "{{ item }}"
+    dest: /usr/local/bin/{{ item }}
+    mode: 0755
+    owner: root
+    group: root
+  with_items:
+    - hub-update
+    - hub-deploy
+    - hub-drush
+  notify:
+    - redeploy containers
```

Comment:
Are we now grabbing containers from docker hub or is this purely designed for reloading?

Reply:
For the staging environment, taking straight from docker hub - yes.

Reply:
Aha, OK.